### PR TITLE
Introduce build-load builds command to measure build registration time

### PR DESCRIPTION
**Pull request by SaschaSchwarze0**
I generalized the `BuildRunResult` into `Result` which is why I had to touch many files. Also updates the client library which caused the mandatory context to be added.

### CODE REVIEW

File: internal/cmd/builds.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
+/*
+Copyright © 2020 The Homeport Team
```
Comment (HeavyWombat) [inline internal/cmd/builds.go:2]: :)

```suggestion
Copyright © 2021 The Homeport Team
```

File: internal/load/common.go [hunk 2]
```diff
@@ -63,6 +63,9 @@ func NewKubeAccess() (*KubeAccess, error) {
 		return nil, err
 	}
 
+	restConfig.QPS = 500
+	restConfig.Burst = 500
```
Comment (HeavyWombat) [inline internal/load/common.go:67]: I am not familiar with those settings. Could you add a comment to explain the reason? Thanks.

File: internal/load/models.go [hunk 3]
```diff
@@ -164,6 +169,16 @@ func createNamespaceAndName(namingCfg NamingConfig, buildCfg BuildConfig, idx in
 	return namingCfg.Namespace, fmt.Sprintf("%s-%s-%d", namingCfg.Prefix, buildCfg.ClusterBuildStrategy, idx)
 }
 
+func createBuildAnnotations(buildCfg BuildConfig) map[string]string {
+	if buildCfg.SkipVerifySourceRepository {
+		annotations := make(map[string]string)
+		annotations[buildv1alpha.AnnotationBuildVerifyRepository] = "false"
+		return annotations
```
Comment (HeavyWombat) [inline internal/load/models.go:176]: Maybe a bit more compact?
```suggestion
		return map[string]string{
			buildv1alpha.AnnotationBuildVerifyRepository: "false",
		}
```

**Review by HeavyWombat — CHANGES_REQUESTED** [review]
Thanks for the PR.

File: internal/load/models.go (hunk 3 above)
Comment (SaschaSchwarze0) [reply internal/load/models.go:176]: Thank you. I was looking for how to use a variable value as key during map initialization, but had not found it. :-)

File: internal/cmd/builds.go (hunk 1 above)
Comment (SaschaSchwarze0) [reply internal/cmd/builds.go]: Done

File: internal/load/common.go (hunk 2 above)
Comment (SaschaSchwarze0) [reply internal/load/common.go:68]: Yeah, I basically wanted to produce many builds at once to put the build controller under pressure. But with the default QPS (5) this was not very efficient. The 500, I arbitrarily selected.

File: internal/load/models.go (hunk 3 above)
Comment (SaschaSchwarze0) [reply internal/load/models.go]: Done.

**Review by HeavyWombat — APPROVED** [review]

**Pull request merged**